### PR TITLE
Deprecate OBJ Clusters List and View Functions

### PR DESCRIPTION
**Pull request by zliang-akamai**
## 📝 Description

This is to deprecate the Object Storage Clusters List and Cluster View API.

Test is not needed because only comments/in-line docs were changed.

### CODE REVIEW

**Comment by lgarber-akamai** [comment]
@zliang-akamai Assuming we're still waiting to deprecate until GA, would it make sense to add a temporary do not merge tag to this PR?

**Comment by zliang-akamai** [comment]
@lgarber-akamai MultiCluster GA is 3 days after LA, so I think we are targeting GA to release everything now.

File: object_storage_clusters.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ func (resp *ObjectStorageClustersPagedResponse) castResult(r *resty.Request, e s
 	return castedRes.Pages, castedRes.Results, nil
 }
 
+// Deprecated: ListObjectStorageClusters is a deprecated API.
```
Comment (lgarber-akamai) [inline object_storage_clusters.go:41]: nitpick:
```suggestion
// Deprecated: ListObjectStorageClusters uses a deprecated API endpoint.
```

File: object_storage_clusters.go [hunk 2]
```diff
@@ -48,6 +49,7 @@ func (c *Client) ListObjectStorageClusters(ctx context.Context, opts *ListOption
 	return response.Data, nil
 }
 
+// Deprecated: GetObjectStorageCluster is a deprecated API.
```
Comment (lgarber-akamai) [inline object_storage_clusters.go:52]: nitpick:
```suggestion
nitpick:
```suggestion
// Deprecated: GetObjectStorageCluster uses a deprecated API endpoint.
```
```

**Review by lgarber-akamai — APPROVED** [review]
LGTM, nice work!

**Review by ykim-1 — APPROVED** [review]
LGTM, Nice work!

**Pull request merged**